airbyte-integrations/connectors/source-instagram/source_instagram/api.py

```diff
@@ -88,7 +88,13 @@ def _find_accounts(self) -> List[Mapping[str, Any]]:
                         }
                     )
         except FacebookRequestError as exc:
```

Comment:
Do all `FacebookRequestError`s we might run into here pertain to permissions issues? Is there something more specific we could except in order to raise this specific error message in order to avoid accidentally masking other issues as permission issues?

Reply:
thanks for commenting, I added improvement for this code, now we checking the error status code and if this code rated to permissions we also say about checking permission needed for our connector.

Reply:
Awesome, thanks!